oddbird/accoutrement
# Modules core

## Cute animal pic
_Because everyone likes pictures of [animals](https://unsplash.com/t/animals)._

![rabbit](https://images.unsplash.com/photo-1627110914363-5ecd867a8037?ixid=MnwxMjA3fDB8MHx0b3BpYy1mZWVkfDJ8SnBnNktpZGwtSGt8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=900&q=60)

## Link to Trello card (if applicable)
_If this PR relates to a Trello card, don't forget to reference this PR on the card as well._

https://trello.com/c/9tyINy2o/23-i-want-accoutrement-on-sass-modules

## Description
_The commit messages say what you did; this should explain why and/or how._

- all the core modules have been split into individual parts: tokens, utils, vars, and ratios
- I tried to document all the changes, but may have missed some?


## Steps to test/reproduce
_Please explain how to best reproduce the issue and/or test the changes locally (including the pages/URLs/views/states to review)._

- run the tests
- compare the overall and sassdoc changelogs to the actual changes made
- check the documentation
